hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/services/AlertsCriteria.java

```diff
@@ -47,15 +47,16 @@
     String triggerId = null;
     Collection<String> triggerIds = null;
     Map<String, String> tags = null;
+    String tagQuery = null;
```

Comment:
Good point.
We certainly we could do that.
I guess we could maintain the old tags query map and convert it into a tagQuery.
The implementation combines mostly of the two approaches, so I don't see issues doing that.
I will take a look at it.